test/deprecated/Math/depExp.chpl

```diff
@@ -0,0 +1,18 @@
+// copied from test/library/standard/Math/exp.chpl
```

Comment:
I might have missed this on previous PRs, but does this test also check the real(32) cases? And for `imag` and `complex`?

Reply:
You're right, it's not (though there isn't an `imag` overload).  I've been relying pretty heavily on pre-existing tests, but we do know that we don't have full test coverage.  Still, the testing system has caught some incorrect adjustments to a smattering of the overloads (like the time I wrote `coz` instead of `cos` for the complex version) so I'm not as worried by it as I could be and am inclined to add it to the growing list of "things that need better testing so we're certain 2.0 is in good shape" that we'll hopefully tackle while playing with the release candidate rather than spending time on the deprecation warning tests themselves.  Does that sound reasonable to you?

Reply:
yes that works for me